HAB5/hab5.tex

```diff
@@ -0,0 +1,404 @@
+It is expected that the payload will be modular, fitting both an x-u form factor and having a power supply of its own. 
+This is because optimization of a HAB vehicle capable of handling any modular payload results in selection for a power system dedicated solely to the HAB vehicle board and merely arming/disarming the payload electronics. 
+Past issues have included adhesives melting, batteries breaking apart and damage due to environmental factors. Each of these contingencies is to be specifically tested and solved for.
+
+\subsubsection{Software and Payload}
+\label{deliverables-payload}
+Python, instead of C/C++ allows for greater accessibility and the tradeoff in moving isn't worth the increased development costs. 
+Previous HAB teams have considered this question, and given the resources on the team, their findings are restated. OpenCV 3 will continue to be used as a major part.
+
+The primary payload utilized is the WUAP2 payload. 
+This is to build on the previous WUAP payload while adding a NDVI system.
+Use of NDVI is to be investigated on the ground and considerable attention paid to how NDVI results may be linked to visual spectrum analysis of vegetation. 
+While specific to the payload mission selected, exploring the linking of different realms of sensor information is a pursuit which will further aid SPEX. 
+An example could be the linking of IMU attitude data to images at the corresponding time, which in turn may allow for correction of apparent areas due to the angle of the camera with respect to the horizon. 
+This in turn prepares SPEX for projects requiring this correction be done on-flight, and for other projects such as horizon detection which could ``train'' on-flight by comparing analyzed data to known IMU readings. 
+To this end the synchronization of clocks may be pursued. 
+Another consistent challenge has been the processing of information on a stream, without information being skipped or resampled. 
+Raw and processed information should be saved for later evaluation and iteration of systems.
+
+Significant departures from prior procedures in this proposition include the rigorous testing of components and assembled hardware before flight, training/tuning of algorithms where needed and the inclusion of metrics to detect components going offline, development and propagation of errors. 
+As SPEX moves to consider CV modules, it is expected that the development of training knowledge in pre-flight testing will become crucial. 
+Among the tests are those for: camera alignment; camera stability; spectral response/quantum efficiency; software processing of simulations; payload NDVI/visual functioning while on ground; an electronics functional test with specific focus on power systems and APRS reliability; electronics environmental testing; integrated system testing.
+Time-stamping of information received by different processes may be considered. 
+It is also desired that metrics be included for post-flight analysis.
```

Comment:
What kind of metrics will be recorded and presented?

Reply:
Time when components go offline or power is lost would be one. APRS packets/times is another. One of the problems with detailing right now is that while quite a few can be added as software, it's hard to decide which can easily be implemented (without causing delays) before a hardware setup is established and running.